contao/contao
# Fix height limit toggler behavior in "select multiple" state

## How to reproduce the issue

* Go to the content elements list view of an article (`do=article&table=tl_content`)
* Insert a content element that is long enough to get height-limited in said view
* Click "edit multiple"
* Click the "limit toggler" button (three dots) to expand the content element

**Result:** The element is expanded, the element's checkbox is checked, and the page is reloaded immediately, showing all elements in collapsed state again

**Expected result:** The element is expanded and nothing else happens: No checkbox select, no reload.

## Cause
Reload: As the "limit toggler" is a button with no type specified, the type defaults to "submit" and the "edit multiple" form is submitted when the button is clicked.

Checkbox toggle: As the button is a child element of `.toggle_select`, the checkbox is toggled whenever the button is clicked.

## Solution
Add type "button" to prevent form submission. Return from click event handling if `.limit_toggler` can be found as target parent.
